### ISSUE TITLE
What is the status of this crate?

### ISSUE THREAD

**Issue by bluthej** [LegNeato/asciicast-rs#6]
Hi,

I was wondering what the status of this crate is? It looks like there hasn't been a commit since August of 2021 so I'm wondering if it's still maintained.

I'm starting a project to port [svg-term](https://github.com/marionebl/svg-term-cli) to Rust and I'm looking at asciicast-rs as a library to depend on!

Cheers

**Comment by LegNeato** [comment]
The format hasn't really changed, so there are no updates needed here minus bugs, dependency updates, or rust updates (asciicast v1 support isn't there, but no one uses it). So the library is supported but not actively developed.

I know the official project has been rewriting a bunch of their stuff in Rust, so you might want to poke around there to see if there is a library for your use case.

**Comment by bluthej** [comment]
Thx for your reply! 

I was also asking to know if contributions were potentially welcome, I noticed that all event types aren't supported for instance (but maybe the ones that aren't are not that useful?).

I did have a look at agg, which is their gif generator in Rust, and it does have an asciicast.rs which corresponds to your crate, but I'd rather depend on a standalone library (plus it looks like agg is not published on crates.io as of right now).
